src/main/java/tasklist/TaskList.java

```diff
@@ -0,0 +1,56 @@
+package tasklist;
+
+import java.util.ArrayList;
+
+import task.Task;
+
+/**
+ * Encapsulates the list of tasks inputted by the user.
+ */
+public class TaskList {
+    public ArrayList<Task> list;
```

Comment:
I would suggest maybe making the ArrayList<Task> list modifier to be private so that only TaskList can access the list! The rest looks good!